### PR TITLE
Pratik

### PR DESCRIPTION
**Pull request by pratiksatham**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -5,42 +5,53 @@
 from flask_bcrypt import bcrypt
 from connection import *
 from queries import *
-from workflow import *
-import datetime
-from datetime import datetime
+from pmt import *
```
Comment (Chandrashekhar-Bhavsar) [inline app.py:8]: Remove unwanated space

File: app.py [hunk 2]
```diff
@@ -5,42 +5,53 @@
...
+#                       authentication module                    #
 ############################################################
+from flask_bcrypt import Bcrypt
+import bcrypt
+bcrypt = Bcrypt(app)
+
+
+@app.route('/login', methods=['POST'])
+def pm_login():
+    return pm_loginn()
```
Comment (Chandrashekhar-Bhavsar) [inline app.py:27]: Function naming should be descriptive

File: pmt.py [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+logging.basicConfig(level=logging.DEBUG)
+
+file = open("myfile.txt","w")
+
+app = Flask(__name__)
+cors = CORS(app)
+
+
+
+##############################################################################################################
```
Comment (Chandrashekhar-Bhavsar) [inline pmt.py:20]: Remove Unwanted Comments

File: pmt.py [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+        values = (Password,)
+        cursor.execute(query, values)
+        users = cursor.fetchone()
+        logging.debug(dt_string,"Password Checking Query executed sucessfully")
+        logging.debug(dt_string,"Query result is ",users)
+        if not users:
+            logging.debug(dt_string,"Password is not valid")
+            return jsonify({'error': 'Password is not valid'}), 400
+        else:
+            flag2=True
```
Comment (Chandrashekhar-Bhavsar) [inline pmt.py:58]: variable name should be functional descriptive

**Review by Chandrashekhar-Bhavsar — COMMENTED** [review]
Resolve the changes

**Review by Simran2711 — APPROVED** [review]

**Review by Chandrashekhar-Bhavsar — APPROVED** [review]

**Review by Chandrashekhar-Bhavsar — APPROVED** [review]

**Review by Chandrashekhar-Bhavsar — APPROVED** [review]

**Pull request closed without merging**